spec/controllers/organizations_controller_spec.rb

```diff
@@ -34,6 +35,34 @@
       end
     end
 
+    context 'user with admin privilege on the organization but not part of the classroom' do
+      before(:each) do
+        organization.users = []
```

Comment:
@johndbritton I thought so too at first, but then I confirmed that this is actually correct in console

Here is my pry terminal

```ruby
>> o = Organization.first
  Organization Load (0.6ms)  SELECT  "organizations".* FROM "organizations" WHERE "organizations"."deleted_at" IS NULL  ORDER BY "organizations"."id" ASC LIMIT 1
=> #<Organization:0x007fc3acf97118
 id: 4,
 github_id: 12439714,
 title: "tatooine-moisture-farmers",
 created_at: Mon, 07 Mar 2016 06:13:25 UTC +00:00,
 updated_at: Mon, 07 Mar 2016 06:13:25 UTC +00:00,
 deleted_at: nil,
 slug: "12439714-tatooine-moisture-farmers">

>> o.users
  User Load (0.6ms)  SELECT "users".* FROM "users" INNER JOIN "organizations_users" ON "users"."id" = "organizations_users"."user_id" WHERE "organizations_users"."organization_id" = $1  [["organization_id", 4]]
=> [#<User:0x007fc3b48ec7c0
  id: 1,
  uid: 564113,
  token: "token-here-nice-try",
  created_at: Sun, 28 Feb 2016 17:17:56 UTC +00:00,
  updated_at: Thu, 17 Mar 2016 03:29:02 UTC +00:00,
  site_admin: true,
  role: 0>]

>> o.users = []
   (0.2ms)  BEGIN
  SQL (0.2ms)  DELETE FROM "organizations_users" WHERE "organizations_users"."organization_id" = $1 AND "organizations_users"."user_id" = 1  [["organization_id", 4]]
   (2.7ms)  COMMIT
=> []

>> Organization.first.users
  Organization Load (0.4ms)  SELECT  "organizations".* FROM "organizations" WHERE "organizations"."deleted_at" IS NULL  ORDER BY "organizations"."id" ASC LIMIT 1
  User Load (0.3ms)  SELECT "users".* FROM "users" INNER JOIN "organizations_users" ON "users"."id" = "organizations_users"."user_id" WHERE "organizations_users"."organization_id" = $1  [["organization_id", 4]]
=> []
```

So this works as expected